### ISSUE TITLE
Not run on Python3.9 

### ISSUE THREAD

**Issue by aishlai** [linode/linode-cli#231]
Traceback (most recent call last):
  File "/usr/local/bin/linode-cli", line 5, in <module>
    from linodecli import main
  File "/usr/local/lib/python3.9/site-packages/linodecli/__init__.py", line 14, in <module>
    from .cli import CLI
  File "/usr/local/lib/python3.9/site-packages/linodecli/cli.py", line 16, in <module>
    from .response import ModelAttr, ResponseModel
  File "/usr/local/lib/python3.9/site-packages/linodecli/response.py", line 7, in <module>
    from colorclass import Color
  File "/usr/local/lib/python3.9/site-packages/colorclass/__init__.py", line 11, in <module>
    from colorclass.codes import list_tags  # noqa
  File "/usr/local/lib/python3.9/site-packages/colorclass/codes.py", line 4, in <module>
    from collections import Mapping
ImportError: cannot import name 'Mapping' from 'collections' (/usr/local/lib/python3.9/collections/__init__.py)

**Comment by Dorthu** [comment]
I could not reproduce this in a python3.9 docker container.  What I did was:

```
$ docker run -ti python:3.9 bash
root@35f96f99f449:/# pip install linode-cli
root@35f96f99f449:/# linode-cli linodes list
```

This worked as expected, accepting a token and then listing my Linodes. 

What command is giving you this error?

Could you also include the output of the following commands?

```
pip show colorclass
pip show linode-cli
```

**Comment by Dorthu** [comment]
It looks like there's a patch for the colorclass library that corrects this import, but I'm not completely sure why it's not working in python 3.9 for you, as the deprecation warning says this:

> DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working

The actual commit that removes `Mapping` from `collections` is [here](https://github.com/python/cpython/commit/c47c78b878ff617164b2b94ff711a6103e781753#diff-c92f871cb9b4aa1583c89f82cd416d58a5d70fff41faf41e45dedf775dbe7713), and is only tagged `v3.10.0a6` and `v3.10.0a5` - I don't believe this should be an issue in 3.9.x.

The upstream fix for colorclass is here: https://github.com/Robpol86/colorclass/pull/27

If you can get me more info on your environment, I'll see if I can reproduce it and figure out what the issue is for 3.9.  Otherwise, if the maintainers of that library won't fix it, I might have to transition to a new library or fork that one.

**Comment by aishlai** [comment]
I have use "update-alternatives" switch to Python3.8, linode-cli is worked, but on the Python3.9.0a1 not work

Python3.9.0a1:
`
➜  ~ python3.9 -V
Python 3.9.0a1

➜  ~ pip3.9 show colorclass
Name: colorclass
Version: 2.2.0
Summary: Colorful worry-free console applications for Linux, Mac OS X, and Windows.
Home-page: https://github.com/Robpol86/colorclass
Author: @Robpol86
Author-email: robpol86@gmail.com
License: MIT
Location: /usr/local/lib/python3.9/site-packages
Requires:
Required-by:

➜  ~ pip3.9 show linode-cli
Name: linode-cli
Version: 5.0.1
Summary: CLI for the Linode API
Home-page: https://www.linode.com/docs/api/
Author: Linode
Author-email: developers@linode.com
License: BSD 3-Clause License
Location: /usr/local/lib/python3.9/site-packages
Requires: requests, terminaltables, PyYAML, colorclass
Required-by:
`
Python3.8
`
➜  ~ python3.8 -V
Python 3.8.5

➜  ~ pip3.8 show linode-cli
Name: linode-cli
Version: 5.0.1
Summary: CLI for the Linode API
Home-page: https://www.linode.com/docs/api/
Author: Linode
Author-email: developers@linode.com
License: BSD 3-Clause License
Location: /usr/local/lib/python3.8/dist-packages
Requires: colorclass, requests, PyYAML, terminaltables
Required-by:

➜  ~ pip3.8 show colorclass
Name: colorclass
Version: 2.2.0
Summary: Colorful worry-free console applications for Linux, Mac OS X, and Windows.
Home-page: https://github.com/Robpol86/colorclass
Author: @Robpol86
Author-email: robpol86@gmail.com
License: MIT
Location: /usr/local/lib/python3.8/dist-packages
Requires:
Required-by: linode-cli
`

**Comment by Dorthu** [comment]
I was able to reproduce this with python 3.9.0a1:

```
$ docker run -ti python:3.9.0a1 bash
root@8807f71bd9b2:/# pip install linode-cli
root@8807f71bd9b2:/# linode-cli linodes list
Traceback (most recent call last):
  File "/usr/local/bin/linode-cli", line 5, in <module>
    from linodecli import main
  File "/usr/local/lib/python3.9/site-packages/linodecli/__init__.py", line 14, in <module>
    from .cli import CLI
  File "/usr/local/lib/python3.9/site-packages/linodecli/cli.py", line 16, in <module>
    from .response import ModelAttr, ResponseModel
  File "/usr/local/lib/python3.9/site-packages/linodecli/response.py", line 7, in <module>
    from colorclass import Color
  File "/usr/local/lib/python3.9/site-packages/colorclass/__init__.py", line 11, in <module>
    from colorclass.codes import list_tags  # noqa
  File "/usr/local/lib/python3.9/site-packages/colorclass/codes.py", line 4, in <module>
    from collections import Mapping
ImportError: cannot import name 'Mapping' from 'collections' (/usr/local/lib/python3.9/collections/__init__.py)
```

According to the [release page](https://www.python.org/downloads/release/python-390a1/), this was an "early developer preview" of python 3.9 from Nov 2019, with the official 3.9.0 being released in Oct 2020.  It looks like this is a bug in the alpha release, and that this feature is not supposed to be removed until 3.10.0.

That said, since the library that is causing this issue appears to be unmaintained, I'm going to leave this issue open and work on replacing it before 3.10.0 comes out in Oct 2021.

**Comment by aishlai** [comment]
> I was able to reproduce this with python 3.9.0a1:
> 
> ```
> $ docker run -ti python:3.9.0a1 bash
> root@8807f71bd9b2:/# pip install linode-cli
> root@8807f71bd9b2:/# linode-cli linodes list
> Traceback (most recent call last):
>   File "/usr/local/bin/linode-cli", line 5, in <module>
>     from linodecli import main
>   File "/usr/local/lib/python3.9/site-packages/linodecli/__init__.py", line 14, in <module>
>     from .cli import CLI
>   File "/usr/local/lib/python3.9/site-packages/linodecli/cli.py", line 16, in <module>
>     from .response import ModelAttr, ResponseModel
>   File "/usr/local/lib/python3.9/site-packages/linodecli/response.py", line 7, in <module>
>     from colorclass import Color
>   File "/usr/local/lib/python3.9/site-packages/colorclass/__init__.py", line 11, in <module>
>     from colorclass.codes import list_tags  # noqa
>   File "/usr/local/lib/python3.9/site-packages/colorclass/codes.py", line 4, in <module>
>     from collections import Mapping
> ImportError: cannot import name 'Mapping' from 'collections' (/usr/local/lib/python3.9/collections/__init__.py)
> ```
> 
> According to the [release page](https://www.python.org/downloads/release/python-390a1/), this was an "early developer preview" of python 3.9 from Nov 2019, with the official 3.9.0 being released in Oct 2020. It looks like this is a bug in the alpha release, and that this feature is not supposed to be removed until 3.10.0.
> 
> That said, since the library that is causing this issue appears to be unmaintained, I'm going to leave this issue open and work on replacing it before 3.10.0 comes out in Oct 2021.

OK thank you very much

**Comment by cmosetick** [comment]
FYI for anyone interested...
I have not had any issues with linode-cli and python 3.9.4.
As previously mentioned, probably something related to an alpha release.

```
python3 --version
Python 3.9.4

pip3 show linode-cli

Name: linode-cli
Version: 5.2.0
Location: /home/linuxbrew/.linuxbrew/lib/python3.9/site-packages
Requires: requests, colorclass, PyYAML, terminaltables
Required-by: 

pip3 show boto      
Name: boto
Version: 2.49.0
Location: /home/linuxbrew/.linuxbrew/lib/python3.9/site-packages
Requires: 
Required-by: 
```
